server/models/remote_provider.go

```diff
@@ -310,7 +311,38 @@ func (l *RemoteProvider) InitiateLogin(w http.ResponseWriter, r *http.Request, _
 			"meshery_version":  []string{mesheryVersion},
 			"ref":              refURL,
 		}
+		if viper.GetBool("PLAYGROUND") {
+			ep, exists := l.Capabilities.GetEndpointForFeature(PersistAnonymousUser)
+			if !exists {
+				l.Log.Warn(ErrInvalidCapability("PersistAnonymousUser", l.ProviderName))
+				return
+			}
+			client := &http.Client{}
+			req, _ := http.NewRequest("GET", l.RemoteProviderURL+ep, nil)
+			req.Header.Set("X-API-Key", GlobalTokenForAnonymousResults)
+			resp, err := client.Do(req)
+			if err != nil {
+				l.Log.Error(ErrDoRequest(err, "GET", l.RemoteProviderURL+ep))
+				return
+			}
+			defer resp.Body.Close()
+			body, err := io.ReadAll(resp.Body)
+			if err != nil {
+				l.Log.Error(ErrDataRead(err, "response body"))
+				return
+			}
 
+			var tokenResp AnonymousTokenResponse
+			err = encoding.Unmarshal(body, &tokenResp)
+			if err != nil {
+				l.Log.Error(err)
+				return
+			}
+			l.SetJWTCookie(w, tokenResp.AccessToken)
+			redirectURL := "/extension/meshmap"
```

Comment:
@Jougan-0 this should not be hardcoded into Meshery Server. This is an extension point. The remote provider needs to provide this information.

Reply:
I agree. In [token handler](https://github.com/meshery/meshery/blob/f079c3696de88a1689ed2b262db3ab80cb54fe16/server/models/remote_provider.go#L3442) we did the same so I used it rather I should have updated both.
I'll update both the places.

Reply:
This has been removed now.